experiment/example/example.go

```diff
@@ -0,0 +1,68 @@
+// Package example contains a simple example of experiment.
+package example
+
+import (
+	"context"
+	"time"
+
+	"github.com/ooni/probe-engine/experiment"
+	"github.com/ooni/probe-engine/experiment/handler"
+	"github.com/ooni/probe-engine/model"
+	"github.com/ooni/probe-engine/session"
+)
+
+const (
+	testName    = "example"
+	testVersion = "0.0.1"
+)
+
+// Config contains the experiment config.
+type Config struct {
+	sleepTime time.Duration
```

Comment:
We want this field to be visible outside this package, hence it must be `SleepTime`.